src/components/home/modals/Send.vue

```diff
@@ -146,7 +146,7 @@
         </b-container>
         <b-container v-if="pageId===2">
           <Confirm :address=showAddress()
-                   :recipient=showRecipient()
+                   :recipient="recipient"
                    :amount=inputAmount(amount)
```

Comment:
Not only address and recipient, but also all other fields need to change to use QRCode `dataObject`.

Reply:
The `dataObject` should generate at step `pageId===2`, and then pass this object through all following steps (pageId 2 to 4).